lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -55,3 +55,22 @@ form.ccx-form {
         margin: 5px 0 5px 0;
     }
 }
+
+button.ccx-button-link {
+    background: none !important;
+    border: none;
+    padding: 0 !important;
```

Comment:
We prefer not to use `!important`. @amir-qayyum-khan if you can provide more specificity in the target here, that'd be preferred.